script.js

```diff
@@ -0,0 +1,66 @@
+const calcRows = document.querySelector('#frame');
+
+let numEntry2 = []
+let numEntry = []
+let operationEntry = []
+let history = []
+
+function recordNumber(event) {
+    if (event.target.className === 'number-button' && operationEntry.length === 0) {
+        numEntry += event.target.innerText
+        console.log(numEntry)
+        return numEntry
+    }
+}
+
+
+function setUpOperator(event) {
+    if (event.target.className === 'operation-button' && numEntry.length > 0) {
+        operationEntry = event.target.innerText
+        let num1 = Number(numEntry)
+        history = [num1, operationEntry]
+        console.log(history)
+        return history
+
+    }
+}
+
+function record2ndNumber(event) {
```

Comment:
would be ideal if you just integrate this into a single recordnumber fn you have on top, ultimately its the same thing, just saving into a differtent place